### DISCUSSION TITLE
Docker container getting bigger and bigger

### DISCUSSION THREAD

**Discussion by Patricen1** [Koenkk/zigbee2mqtt#23195] [General]
Hello,
Z2M is running in a docker container, and container size is increasing up to the available memory left (at the moment 24GB).
Is there a way to find out what is the file that increases within docker container and how to prevent the specified file to increase (I do suspect a log file).

**Comment by Patricen1** [comment]
Is there a way to delete the container and create it back without deleting conficuration?

**Comment by Patricen1** [comment]
I finally deleted and rebuilt container, that solved the size issue.
There is still a stability issue

**Comment by SpangleLabs** [comment]
I've witnessed this same thing btw, my zigbee2mqtt container has grown to 70GB
I can't see why, if I `docker exec` into the container and run `du`, it claims the space isn't taken up
```
# du -d1 -h / | sort -h
[...]
78.7M   /usr
310.3M  /app
393.3M  /
```
But the docker image is definitely 70GB
```
$ du -d1 -h /var/lib/docker/containers | sort -h
[...]
70G     /var/lib/docker/containers/dbf2a867f04903a31e48d656b3edafe95d60eb61c9c0f755b17401f2fc38caf1
71G     /var/lib/docker/containers
$ docker ps | grep dbf
dbf2a867f049   koenkk/zigbee2mqtt                                  "docker-entrypoint.s…"   8 weeks ago   Up 20 hours            8080/tcp, 0.0.0.0:9077->9077/tcp, [::]:9077->9077/tcp                                                                                                                                                                                     zigbee2mqtt
```
